_sub/compute/helm-crossplane/main.tf

```diff
@@ -0,0 +1,23 @@
+locals {
+  packages_list = replace(yamlencode({provider:{packages:var.crossplane_providers}}), "\"", "")
+}
+resource "helm_release" "crossplane" {
+    name = var.release_name
+    chart = "crossplane"
+    repository    = "https://charts.crossplane.io/stable"
+    version       = var.chart_version != null ? var.chart_version : null

```

Comment:
If the default is null do wee need this logic?

Reply:
I guess not, I'll remove it